doc/nrf-bm/app_dev/dfu/memory_partitioning.rst

```diff
@@ -106,3 +106,48 @@ The placement of the SoftDevice in memory is predetermined and must align with s
 - **SD_StartAddr**: Must align on a 2 kB memory boundary as specified in the SoftDevice release notes.
 
 The size of the SoftDevice is adjusted taking into account the above alignment and space reservations.
+
+Partitions sizes reference
+**************************
+
+The following table shows the recommended sizes for each partition when using the Single-Bank Device Firmware Update (DFU) Process.
+
++--------------------------+----------------------------------------------------------+--------------------+-------------------------+
+| Partition Name           | Content                                                  | Recomended size    | Minimum size            |
+|                          |                                                          | (Development)      | (Release)               |
++==========================+==========================================================+====================+=========================+
+| ``boot_partition``       | MCUboot                                                  | 31 KiB             | 21 KiB (using KMU)      |
+|                          |                                                          |                    | 26 KiB (without KMU)    |
++--------------------------+----------------------------------------------------------+--------------------+-------------------------+
+| ``slot0_partition``      | Main application                                         |                    |                         |
++--------------------------+----------------------------------------------------------+--------------------+-------------------------+
+| ``slot1_partition``      | Firmware loader                                          | 48 KiB (Minimum)   | 28 KiB                  |
```

Comment:
You list both 28kB and 32kB in this line. How do you expect customer to read this?
(When I try the recommended  sample I get 28.900 bytes that is higher than 28 kB)
Should we instead only list the recommended numbers and have a more descriptive text below for the different lines?